frontend/src/pages/order/components/RightSideNavbar.tsx

```diff
@@ -0,0 +1,61 @@
+import { Link } from 'react-router-dom'
+import { AppRoutes } from '../../../App'
+import { BsPersonCircle } from 'react-icons/all'
+import styled from 'styled-components'
+import { T } from '../../../reusable-ui/design-system'
+
+export function RightSideNavbar(props: { firstName: any }) {
+  return (
+    <RightSideNavbarStyled className={'rightSide'}>
+      <div className={'textContainer'}>
+        <p className={'welcomePhrase'}>
+          Hey, <span>{props.firstName}</span>
```

Comment:
```suggestion
          Hey, <span>{firstName}</span>
```

De préférence, destructurer "props" pour extraire d'ores et déjà "les props". C'est plus léger à la lecture et à l'utilisation dans le reste du composant, pour soi et pour ses collègues.